### ISSUE TITLE
Training loss decreases while validation loss does not; might be because of overfitting

### ISSUE THREAD

**Issue by nos1dot618** [Hyouteki/BTP#6]
Reference: https://github.com/keras-team/keras/issues/8471

> Training loss decreases while validation loss does not and same for accuracy; training accuracy increases while validation accuracy does not grow with the same pace.

![loss over epoch](https://github.com/Hyouteki/btp/assets/108230497/6507440b-5271-4bb7-91a0-87c87dc1fbb8)
![acc over epoch](https://github.com/Hyouteki/btp/assets/108230497/a5f08b4d-4e17-4d60-884a-f8d711be555b)
